<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QMainWindow" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>349</width>
    <height>385</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>349</width>
    <height>385</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fenetre</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>312</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="indication">
       <property name="minimumSize">
        <size>
         <width>303</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Veuillez vous connecter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="espace1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="loginLabel">
       <property name="text">
        <string>Votre identifiant</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="LoginEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblMdp">
       <property name="text">
        <string>Votre mot de passe</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="MdpEdit">
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="boutonConnect">
       <property name="text">
        <string>Se connecter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="avertissement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAutres">
    <property name="title">
     <string>Aut&amp;res</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuAutres"/>
  </widget>
  <action name="apropos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>&amp;A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
